packages/web-comment/src/server.config.js

```diff
@@ -2,7 +2,8 @@ const httpPort = Number(process.env.PORT || 3000);
 
 module.exports = {
 	appealsApi: {
-		baseUrl: process.env.APPEALS_SERVICE_API_URL || `http://localhost:3000`
+		baseUrl: process.env.APPEALS_SERVICE_API_URL || `http://localhost:3000`,
+		timeout: Number(process.env.APPEALS_SERVICE_API_TIMEOUT || 10000)
```

Comment:
know this is copied, but should we use parse int instead?

Reply:
Yeah good idea: https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/3359/commits/0667660b4601de38876a3513ed3263900270de97